solidity/dashboard/src/components/BeaconRewardsDetails.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import * as Icons from "./Icons"
+import BeaconRewardsHelper from "../utils/beaconRewardsHelper"
+// import { displayAmount } from "../utils/token.utils"
+import { formatDate } from "../utils/general.utils"
+
+const BeaconRewardsDetails = ({ pastRewards }) => {
+  const intervalsLeft =
+    BeaconRewardsHelper.keepAllocationsInInterval.length -
+    BeaconRewardsHelper.currentInterval -
+    1
+
+  const nextIntervalStart = formatDate(
+    BeaconRewardsHelper.intervalStartOf(BeaconRewardsHelper.currentInterval + 1)
+  )
+
+  return (
+    <>
+      <h4 className="text-grey-70 mb-2">Beacon Rewards Details</h4>
+      <ul>
+        <li className="flex row center mb-1">
+          <Icons.Beacon width={12} height={12} />
+          <span className="text-small ml-1">Min. Keep Group</span>
```

Comment:
I think this can be “Minimum required groups this interval”, or just “Min. Groups”.